containers/Filters/AddFilterModal.css

```diff
@@ -0,0 +1,3 @@
+.AddFilterModal-isPreview .AddFilterModal-editor {
```

Comment:
Why not using a ternary condition inside the React component?

Reply:
Because it's easier to do it once ;) on the container